src/datatypes/datetime.ts

```diff
@@ -99,7 +99,7 @@ export class LocalDateTime {
   }
 
   toString(): string {
-    return this._date.toUTCString();
+    return this._date.toUTCString().slice(0, -4); // cut off " GMT"
```

Comment:
Does it always end with "GMT"? Maybe we want to add an assertion here?

Reply:
OK, I've added a check.